client/material/overlay.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
```

Comment:
Similar to above, I would move this file to the `profile` folder as well and perhaps rename it to `profile-overlay` or something.

Perhaps I misled you a bit when I said that this should be a general component. I meant that it should be a general component for the two profile overlays that we have now, not general for the whole application ^^. The popover component that we have in the material folder is already the general one that allows configuring transition timings and whatnot.

If we decide in the future that the transition timings that we're using here for the profile overlays want to also use in other places, unrelated to the profile overlays, then we can think about moving it to a more general place like material folder (which is what we're doing for the `Menu` component for example).